README.md

```diff
@@ -20,9 +20,11 @@ sudo add-apt-repository ppa:deadsnakes/ppa
 sudo apt-get install python3.12 python3.12-dev python3.12-venv python3.12-tk
 ```
 
+:warning: We highly suggest to install uv to speedup scilpy installation: https://docs.astral.sh/uv/getting-started/installation/
```

Comment:
Since `uv` is optional, I think we should mention that `scilpy` can still be installed by omitting the `uv` from all the installation command lines below.